lib/thesis/view.ex

```diff
@@ -193,16 +193,21 @@ defmodule Thesis.View do
     raw """
     <script>
       ;(function () {
-        var loadThesis = function () {
+        var loadThesis = function (callback) {
```

Comment:
don't use `var` use `let` or even better!  `const`! - so that you get a more functional feel.